isisd/isis_spf.c

```diff
@@ -1261,6 +1273,11 @@ isis_run_spf_l1 (struct thread *thread)
   area = THREAD_ARG (thread);
   assert (area);
 
+  if (area->spf_delay_ietf[0]) {
+    area->spf_delay_ietf[0]->family[0]->t_spf = NULL;
+    area->spf_delay_ietf[0]->family[0]->t_spf = NULL;
```

Comment:
That's a mistake, there should be a single line as this function only process ipv4 level 1.
The other occurences are coming from copy/paste and should also be fixed by removing the other line.